interface/themes/style_ash_blue.css

```diff
@@ -1,7 +1,7 @@
 /**
```

Comment:
brought this script changes into codebase
(note I removed the end of line whitespaced)